packages/angular/angular-sdk/src/lib/sdk/space.service.ts

```
@@ -1,14 +1,120 @@
+  private spaceInitializedSubject = new BehaviorSubject<boolean>(false)
+  public spaceInitialized$: Observable<boolean> =
+    this.spaceInitializedSubject.asObservable()
+  get spaceInitialized(): boolean {
+    return this.spaceInitializedSubject.value
+  }
+  set spaceInitialized(value: boolean) {
+    this.spaceInitializedSubject.next(value)
+  }
+  public spaceResponse: any
 
   constructor() {}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)